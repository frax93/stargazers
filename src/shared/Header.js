import { StyleSheet, Text } from 'react-native';


export default function Header() {
  return (
    <Text style={styles.title}>Stargazers</Text>
  );
}

const styles = StyleSheet.create({
  title: {
    fontWeight: 'bold',
    textAlign: 'center',
    color: 'white',
    fontSize: 22
  },
});
