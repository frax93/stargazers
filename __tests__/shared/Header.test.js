import Header from 'shared/Header';
import { render } from '@testing-library/react-native';

test('renders the Header', () => {
  const headerTree = render(<Header />).toJSON();
  expect(headerTree).toMatchSnapshot();
});
