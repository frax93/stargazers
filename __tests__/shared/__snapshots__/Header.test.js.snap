// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Header 1`] = `
<Text
  style={
    {
      "color": "white",
      "fontSize": 22,
      "fontWeight": "bold",
      "textAlign": "center",
    }
  }
>
  Stargazers
</Text>
`;
